Xamarin.Forms.Platform.UAP/NavigationPageRenderer.cs

```diff
@@ -419,7 +419,7 @@ void OnPointerPressed(object sender, PointerRoutedEventArgs e)
 
 		void OnPopRequested(object sender, NavigationRequestedEventArgs e)
 		{
-			var newCurrent = Element.Peek(0);
+			var newCurrent = Element.Peek(1);
```

Comment:
Why, is different from UWP ?

Reply:
Element.Peek(0) was my workaround to navigate to the page. 
but by default in UWP it's Element.Peek(1).
It's was before I understood properly the issue.